style.css

```diff
@@ -42,3 +42,17 @@
   background: #777 !important;
 }
 
+/* Items in the left navigation menu */
+.navigation-item .navigation-item-level-0 {
+  color: grey !important;
```

Comment:
please use the hex letter codes in this file so that everything is consistent. e.g. #333 etc.